mmv1/templates/terraform/metadata.yaml.tmpl

```
@@ -3,3 +3,14 @@ generation_type: 'mmv1'
 api_service_name: '{{ $.ProductMetadata.ServiceName }}'
 api_version: '{{ or $.ProductMetadata.ServiceVersion $.ServiceVersion }}'
 api_resource_type_kind: '{{ or $.ApiResourceTypeKind $.Name }}'
+fields:
+{{- range $p := $.LeafProperties }}
+  -
+    field: '{{ $p.MetadataLineage }}'
```

I think this newline isn't necessary / isn't how we generally format our yaml. It adds a lot of whitespace to the yaml file.

```suggestion
  - field: '{{ $p.MetadataLineage }}'
```

---

Thanks yea, that's much better